python/tests/test_umap.py

```diff
@@ -102,7 +101,7 @@ def _spark_umap_trustworthiness(
             )
 
         data_df = data_df.repartition(n_parts)
-        umap_estimator.setFeaturesCol(features_col).setSampleFraction(sampling_ratio)
```

Comment:
Is there still some test of the sample fraction param?

Reply:
Added test